### ISSUE TITLE
[Keras] Support multiple validation sets in Model.fit

### ISSUE THREAD

**Issue by reuben** [tensorflow/tensorflow#38803]
**System information**
- TensorFlow version (you are using): 2.2-rc3
- Are you willing to contribute it (Yes/No): No

**Describe the feature and the current behavior/state.**
Currently there's no way to use multiple validation sets with independent tracking of metrics.

**Will this change the current api? How?**

Simplest way I can think of is to accept a list of datasets in the `validation_data` parameter in `Model.fit`. Ideally there should also be a way to specify the name of each set so that the logs indicate what set each validation step corresponds to.

**Who will benefit with this feature?**

Anyone training with multiple validation sets.

**Comment by fcakyon** [comment]
I really need that feature :)
I'm putting this as a reference https://stackoverflow.com/a/47738812

**Comment by swghosh** [comment]
Hi, Is this feature request still active?

**Comment by yolpsoftware** [comment]
This would be nice. I often have several validation sets with varying difficulty. I expect the model to get nearly 100% accuracy on the easiest validation set. If it does not achieve a high accuracy there, there might be a problem in the setup.

On the most difficult validation set however, you cannot expect high accuracy.

With just one validation set, this difference is not visible.

**Comment by swghosh** [comment]
@yolpsoftware 
Probably, as a workaround for now you can use a custom callback which `model.evaluate`(s) your extra validation set `on_epoch_end`. 
An example can be found here: https://github.com/tanzhenyu/image_augmentation/blob/master/image_augmentation/callbacks/extra_eval.py

**Comment by SergioG-M** [comment]
Any update on this? If I try a custom callback I get the issue mentioned in [#43262](https://github.com/tensorflow/tensorflow/issues/43262). The only workaround I see is use the callback without validation_data in fit

**Comment by jvishnuvardhan** [comment]
@reuben Is there any actionable item like raising PR? Can you please move this to keras-team/keras repo?

Please note that Keras development moved to another repository to focus entirely on only keras. Could you please repost this issue on [keras-team/keras repo](https://github.com/keras-team/keras/issues). Thanks!

**Comment by reuben** [comment]
Feel free to move/repost at will. All due respect, but you chose to shuffle things around ;)

**Comment by jvishnuvardhan** [comment]
Let's track this feature request in keras-team/keras repo https://github.com/keras-team/keras/issues/15757
